src/tokenizer/index.ts

```diff
@@ -12,7 +12,11 @@ export default class Tokenizer {
     this.parser = new MarkdownIt(config);
     this.parser.use(annotations, 'annotations', {});
     this.parser.use(frontmatter, 'frontmatter', {});
-    this.parser.disable('lheading');
+    this.parser.disable([
+      'lheading',
+      // Disable indented `code_block` support https://spec.commonmark.org/0.30/#indented-code-block
+      'code',
```

Comment:
does this disable regular fence code blocks too?

<img width="1070" alt="image" src="https://user-images.githubusercontent.com/55463370/175605738-94abccf2-6c4d-4bc3-a98d-5b370ea953b9.png">

Reply:
Fences are supported by this rule: https://github.com/markdown-it/markdown-it/blob/master/lib/rules_block/fence.js

`code_block` nodes are supported by this rule (with name: `code`): https://github.com/markdown-it/markdown-it/blob/master/lib/rules_block/code.js

Reply:
huh. That is confusing 😆